test/edu-benefits/1995/config/veteranInformation.unit.spec.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1995/config/form';
+
+describe('Edu 1995 veteranInformation', () => {
+  const { schema, uiSchema } = formConfig.chapters.veteranInformation.pages.veteranInformation;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    expect(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input'))
+      .to.not.be.empty;
+  });
+  it('should conditionally require file number', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          formData={{}}
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    ReactTestUtils.findRenderedComponentWithType(form, Form).onSubmit({
+      preventDefault: f => f
+    });
+
+    // show error for ssn, file number not visible
+    expect(formDOM.querySelector('#root_vaFileNumber')).to.be.null;
+    expect(formDOM.querySelector('.usa-input-error #root_veteranSocialSecurityNumber')).not.to.be.null;
+
+    ReactTestUtils.Simulate.change(ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input')[4],
```

Comment:
This could be hard to maintain--maybe add a comment that veteranInformation has a fullName (4 inputs), followed by an SSN input, or just search for it by name if possible.